iot-hub/Samples/service/FileUploadNotificationReceiverSample/FileUploadNotificationReceiverSample.csproj

```diff
@@ -3,12 +3,12 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
-    <LangVersion>8</LangVersion>
+    <LangVersion>9.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Devices" Version="1.36.0" />
-    <PackageReference Include="CommandLineParser" Version="2.8.0" />
+    <PackageReference Include="CommandLineParser" Version="2.9.1" />
```

Comment:
Is there a good way to factor this out into a variable in .NET? In Java, we have all dependency versions like these centralized and it has made dependency management much easier

Reply:
https://github.com/Azure/azure-iot-sdk-java/blob/edb759adc76972b407cd64ab6aae0549895af094/pom.xml#L22

Reply:
It is possible, but makes the samples less readable and then it doesn't work with the NuGet package manager anymore. Not worth it, IMHO.